chapters/annotations.tex

```diff
@@ -888,6 +888,14 @@ \subsection{Component Instance}\label{component-instance}
 Sub-connectors in a hierarchical connector are only shown when they can be connected to.
 Non-connector components are only shown in the diagram layer.
 
+\subsubsection{Default Outline for Missing Graphics}\label{default-outline-for-missing-graphics}
+If the icon of a component placed in a diagram layer does not contain any graphical primitives (including inherited ones, and regardless of \lstinline!visible!-attributes; but excluding connectors), tools shall show a tool-dependent rudimentary outline of the component's transformed \lstinline!extent!.
+
+\begin{nonnormative}
+The reason for making the tool-dependent outline rudimentary is to encourage the model developer to provide a proper icon.
+Regardless of \lstinline!visible!-attributes imply that all primitives are considered regardless of the value of their \lstinline!visible!-attribute.
```

Comment:
Now that the normative rule is formulated in terms of the `visible`-attribute, I find this explanation redundant:
```suggestion
```